app/controllers/posts_controller.rb

```diff
@@ -5,5 +5,23 @@ def index
 
   def show
     @post = Post.find(params[:id])
+    @current = current_user
+  end
+
+  def new
+    @user = current_user
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
```

Comment:
## Optional
- [ ] I suggest that you use `the strong parameters` and remember this method should be private
```ruby
 def post_params
      params.require(:post).permit(:title, :text)
    end
```